src/modules/projects/components/tables/ProjectSubmissionsTable.tsx

```diff
@@ -0,0 +1,80 @@
+import { Chip } from '@mui/material';
+import { capitalize } from 'lodash-es';
+import { useCallback } from 'react';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import {
+  getCustomDataElementColumns,
+  parseAndFormatDate,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ExternalFormSubmissionFieldsFragment,
+  ExternalFormSubmissionFilterOptions,
+  ExternalFormSubmissionStatus,
+  GetProjectExternalFormSubmissionsDocument,
+  GetProjectExternalFormSubmissionsQuery,
+  GetProjectExternalFormSubmissionsQueryVariables,
+  PickListOption,
+} from '@/types/gqlTypes';
+
+export type ExternalFormSubmissionFields = NonNullable<
+  GetProjectExternalFormSubmissionsQuery['project']
+>['externalFormSubmissions']['nodes'][number];
+
+const ProjectSubmissionsTable = ({
+  projectId,
+  formType,
+}: {
+  projectId: string;
+  formType: PickListOption;
```

Comment:
Small nit: I think it's clearer to accept `formIdentifier: string` since that's more descriptive of what is needed.